docs/source/markdown/podman-system-prune.1.md

```diff
@@ -1,7 +1,7 @@
 % podman-system-prune(1)
 
 ## NAME
-podman\-system\-prune - Remove all unused pod, container, image and volume data
+podman\-system\-prune - Remove all unused pods, containers, images, networks and volume data
```

Comment:
```suggestion
podman\-system\-prune - Remove all unused pods, containers, images, networks, and volume data
```

Reply:
I add commas and periods to the NAME section of 'podman-system-prune.1.md' and the subcommand description section of 'podman-system.1.md'.

However, the period in the NAME section of 'podman-system-prune.1.md' is ignored and the validation test failed.

Therefore, I do not add the period to the NAME section.